### PR TITLE
[WIP] switch to ember-cp-validations

### PR DESCRIPTION
**Pull request by marcoow**
This is only a POC for now but I think ember-cp-validations might be the better solution than ember-validations for the long term.

/cc @LevelbossMike

### CODE REVIEW

**Comment by LevelbossMike** [comment]
:+1: for changing to ember-cp-validations. The computed idea definitely makes sense and it looks like a clean implementation from what I saw in the ember global meetup presentation

**Comment by marcoow** [comment]
Yeah, will clean this up and merge it.

File: addon/form-object-cp.js [hunk 1]
```diff
@@ -1,10 +1,15 @@
 import Ember from 'ember';
 import FormObject from './form-object';
 
-export default function formObject(model, impl = {}) {
-  return Ember.computed(model, function() {
-    return FormObject.extend(impl).create({
-      content: this.get(model)
+export default function formObject(modelProperty, Validations = {}, impl = {}) {
```
Comment (oliverbarnes) [inline addon/form-object-cp.js]: `impl`?

Comment (marcoow) [inline addon/form-object-cp.js]: for "implementation" - additional methods, properties etc. that are defined on the created buffer.

**Pull request merged**